mne/annotations.py

```diff
@@ -736,8 +736,14 @@ def is_orig_time(x):
 
 
 def _read_annotations_txt(fname):
-    onset, duration, desc = np.loadtxt(fname, delimiter=',',
-                                       dtype=np.bytes_, unpack=True)
+    import warnings
+    warnings.filterwarnings("ignore", message=('loadtxt: Empty input file: "%s"' % fname))
```

Comment:
Put this in a context manager so that its effect is temporary:
```
try:
    with warnings.catch_warnings(record=True):
        warnings.simplefilter(...)
        onset, duration, ...
```
Move the `import warnings` to the top of the file